tests/unit/net/filemanager_thread.rs

```diff
@@ -56,8 +56,27 @@ fn test_filemanager() {
 
             let msg = rx2.recv().expect("Broken channel");
 
-            let blob_buf = msg.expect("File manager reading failure is unexpected");
-            assert_eq!(test_file_content, blob_buf.bytes, "Read content differs");
+            if let ReadFileProgress::Meta(blob_buf) = msg.expect("File manager reading failure is unexpected") {
+                let mut bytes = blob_buf.bytes;
+
+                loop {
+                    match rx2.recv().expect("Broken channel").expect("File manager reading failure is unexpected") {
+                        ReadFileProgress::Meta(_) => {
+                            panic!("Invalid FileManager reply");
+                        }
+                        ReadFileProgress::Partial(mut bytes_in) => {
+                            bytes.append(&mut bytes_in);
```

Comment:
Okay I see. In [`chunked_read`](https://github.com/izgzhen/servo/commit/98e039f2867f22d14cb83470d42ad85a16cfa38f), `read` is used, which I suppose means that, the length of real chunk we read might be different depending on the system status.